pennylane/pauli/utils.py

```diff
@@ -698,6 +671,26 @@ def is_qwc(pauli_vec_1, pauli_vec_2):
     return True
 
 
+def _are_pauli_words_qwc_pauli_rep(lst_pauli_words):
+    """Given a list of observables assumed to be valid Pauli words, determine if they are pairwise
```

Comment:
Since the ops have to be Operators and not actual PauliWords this sentence doesnt really work

Reply:
This is a situation of ambiguity between `PauliWord`s and Pauli words. I'll update it to say "Pauli observables" instead.